packages/babel-traverse/src/path/interface-compatibility.js

```diff
@@ -0,0 +1,77 @@
+let deprecateShown = false;
+function showDeprecationWarning() {
+  if (!deprecateShown) {
+    console.trace(
+      "Accessing NodePath.skipKeys is deprecated, use NodePath.skipKey(key) instead",
+    );
+  }
+  deprecateShown = true;
+}
+
+export function generateSkipKeysGetter(skipKeysSet) {
+  return new Proxy(
```

Comment:
Can we cache this proxy, to avoid recreating it every time that `skipKeys` is read?